tracer/test/benchmarks/Benchmarks.Trace/AgentFilterAttribute.cs

```diff
@@ -63,6 +77,8 @@ public enum Agent
             BenchmarkAgent5,
             BenchmarkAgent6,
             BenchmarkAgent7,
+            BenchmarkAgent8,
+            BenchmarkAgent9
```

Comment:
Have you _provisioned_ these additional agents? Without doing so, the CI will fail.

Reply:
no applicable now